master/internal/command/command.go

```diff
@@ -282,8 +285,14 @@ func (c *command) handleAPIRequest(ctx *actor.Context, apiCtx echo.Context) {
 }
 
 func (c *command) terminate(ctx *actor.Context) {
-	ctx.Ask(c.rps, scheduler.TerminateTask{TaskID: c.taskID, Forcible: true}).Get()
-	if msg, ok := ctx.Message().(scheduler.TerminateRequest); ok {
+	ctx.Log().Info("terminating")
+	ctx.Tell(c.rps, scheduler.ResourceReleased{Handler: ctx.Self()})
```

Comment:
Fixed that by ignoring the allocated messages if the allocated messages use different task IDs as the trial and the command currently use

Reply:
talked offline. Rather than sending `ResourcesReleased` right after killing the containers, we do that when receving the container state changed messages.